src/main/sass/_page-navigation.scss

```diff
@@ -0,0 +1,105 @@
+.previous-next-navigation {
```

Comment:
spoke to @dsingleton who said that pulling things in from govuk-component might not be the best idea as it's very specific to GOV.UK's own needs.

If there were a navigation widget in govuk-element it might have been nice to use it, but there isn't.

So, this is fine for now.

ping @robinwhittleton who might be interested in this discussion